settings.gradle.kts

```diff
@@ -10,22 +10,47 @@ pluginManagement {
 }
 
 plugins {
-    id("com.gradle.enterprise") version ("3.14.1")
-    id("com.gradle.enterprise.gradle-enterprise-conventions-plugin") version ("0.7.4")
-    id("com.gradle.common-custom-user-data-gradle-plugin") version ("1.11.1")
+    id("com.gradle.develocity").version("3.17.6")
+    id("com.gradle.common-custom-user-data-gradle-plugin") version "2.0.2"
 }
 
 dependencyResolutionManagement {
     repositories {
         mavenCentral()
     }
 }
+
 val isCI = System.getenv("CI")?.toBoolean() ?: false
 val isCC = gradle.serviceOf<BuildFeatures>().configurationCache.active.getOrElse(false)
 
 require(!isCC || !isCI) { "Configuration-Cache should be disabled on CI" }
 
+develocity {
+    server = "https://ge.gradle.org"
+    buildScan {
+        uploadInBackground = !isCI
+        publishing.onlyIf { it.isAuthenticated }
+        obfuscation {
+            ipAddresses { addresses -> addresses.map { "0.0.0.0" } }
+        }
+    }
+}
+
+buildCache {
+    local {
+        isEnabled = true
+    }
+
+    remote(develocity.buildCache) {
+        server = "https://eu-build-cache.gradle.org"
```

Comment:
why not the default?

Reply:
See [thread](https://gradle.slack.com/archives/C032LFSKY/p1722411224756959). The default for ge.gradle.org is not used and eu node was the default till now, so that was the decision (to leave it as default).